### PR TITLE
Add notifications view

### PR DESCRIPTION
**Pull request by tenshiemi**
This adds the view for notifications, it doesn't have content yet. It also adds a new menu for admin users. Right now it displays if you are logged in, but we can update the logic easily later.

### CODE REVIEW

File: client/styles/custom.scss [hunk 1]
```diff
@@ -55,3 +55,7 @@ $tablet: (
   clear: both;
   height: 0;
 }
+
+.hide {
+  display: none !important;
```
Comment (macrael) [inline client/styles/custom.scss:60]: Wooo!

File: client/src/header/NavMenu.js [hunk 2]
```diff
@@ -1,42 +1,63 @@
...
-const NavMenu = ({ loggedIn, logOutUser, userEmail }) => (
-  <div className="container--content">
-    <div className="usa-navbar">
-      <button className="usa-menu-btn">Menu</button>
-      <div className="usa-logo" id="logo">
-        <em className="usa-logo-text">
-          <a href="" title="Home" aria-label="Home">Emergency Alert App</a>
-        </em>
+const NavMenu = ({ loggedIn, logOutUser, userEmail }) => {
+  const showAdmin = classnames({
```
Comment (macrael) [inline client/src/header/NavMenu.js:6]: What does classnames do?

File: client/src/admin/NotificationsPage.js [hunk 3]
```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+const NotificationsPage = () => (
+  <div className="container--content">
```
Comment (macrael) [inline client/src/admin/NotificationsPage.js:4]: While this does technically add a notifications page, why do we not have anything actually displaying for notifications? Even though the end point doesn't exist yet I'm hesitant to merge this as a dummy page.

**Review by macrael — CHANGES_REQUESTED** [review]

File: client/src/admin/NotificationsPage.js (hunk 3 above)
Comment (tenshiemi) [reply client/src/admin/NotificationsPage.js:4]: I can put some temp content in it if you prefer I'd rather do it in a separate branch.

File: client/src/header/NavMenu.js (hunk 2 above)
Comment (tenshiemi) [reply client/src/header/NavMenu.js:6]: Classnames is for generating conditional class sets

File: client/styles/custom.scss (hunk 1 above)
Comment (tenshiemi) [reply client/styles/custom.scss:60]: This was the only way I could get the USWDS conditional menus to work unfortunately -_- Displaying them using ternary statements broke the js.

Comment (macrael) [reply client/styles/custom.scss:60]: It's totally fine for a prototype. So often it's the choice we have to make with CSS

File: client/src/admin/NotificationsPage.js (hunk 3 above)
Comment (macrael) [reply client/src/admin/NotificationsPage.js:4]: Why temp content? Shouldn't we design what the table is going to look like? For all our other pages we put together the UI before we had a server. We should do the same here. I suggest creating a fake server response and using that to populate real views.

If you'd like that to be another PR that's fine. This one looks good as is.

**Review by macrael — APPROVED** [review]

**Pull request merged**